### ISSUE TITLE
Windows automatic update prompts "Application is running

### ISSUE THREAD

**Issue by ozooner** [electron-userland/electron-builder#2014]
I recently updated from packager **19.18.0** to latest **19.27.0**
We have an app in production packaged with 19.18.0, and when I build an update with latest version, windows automatic update displays a prompt:
![error](https://user-images.githubusercontent.com/2709373/29879794-c62bff52-8dae-11e7-86c9-ad6dfaa04561.png)
While pressing OK works fine, this is issue for unmanaged setups.
Building the update with builder v19.18.0 installs without any prompts.

Build options:
target: **nsis 32bit**
provider: generic
oneClick: true
perMachine: false

**Comment by develar** [comment]
Do you use automatic installation on quit or explicit? If explicit, do you use setImmediate?

**Comment by ozooner** [comment]
I call updater like so:
```
autoUpdater.on('update-downloaded', (info) => {
  log.info("Update downloaded, exiting application to install update...");
  autoUpdater.quitAndInstall();                                                                                                                                                                               
});
```
I don't call setImmidiate, but I didn't call it with version 19.18.0 either, and it works there...

**Comment by develar** [comment]
Developer info: probably regression after migration to `StdUtils.TestParameter`

**Comment by develar** [comment]
Thanks a lot for the but report.

**Comment by develar** [comment]
Fixed in 19.27.1.

**Comment by ozooner** [comment]
Wow, that is insanely fast. I'll have one what you're having for motivation :laughing: 
Thanks!
